secure/key-manager/src/counters.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use libra_secure_push_metrics::{define_counters, Counter};
+use once_cell::sync::Lazy;
+use std::sync::Arc;
+
+// Metrics for the key manager. Uses the "libra_key_manager" prefix for all counters.
+define_counters![
+    "libra_key_manager",
+    (
+        full_consensus_rotations: Counter,
+        "counts the number of full consensus key rotations performed by the key manager"
+    ),
+    (
+        rotation_transaction_resubmissions: Counter,
+        "counts the number of times the key manager had to resubmit a rotation transaction to the blockchain"
```

Comment:
is this per individual key or total across all time?

Reply:
I'd count submissions and not resubmissions, again per-key type:

    libra_key_manager_transaction_submits{key="consensus"}

Reply:
This is for the consensus key only (over all execution time). To avoid confusion I've updated the name to be consensus key specific: "consensus_rotation_tx_resubmissions". If/when network key rotation is supported, we can prefix each relevant metric name with the key name.